fullscreen/api/element-request-fullscreen-dialog.html

```diff
@@ -2,15 +2,17 @@
 <title>Element#requestFullscreen() for dialog element</title>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<script src="../trusted-click.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
 <div id="log"></div>
 <dialog></dialog>
 <script>
 async_test(t => {
   var dialog = document.querySelector("dialog");
   assert_true(dialog instanceof HTMLDialogElement);
-  trusted_request(t, dialog);
   document.onfullscreenchange = t.unreached_func("fullscreenchange event");
   document.onfullscreenerror = t.step_func_done();
+
+  test_driver.bless("Initiate fullscreen", () => dialog.requestFullscreen());
```

Comment:
```suggestion
  test_driver.bless("Initiate fullscreen", () => dialog.requestFullscreen().catch(() => {});
```

Looking at [the results](https://wpt.fyi/results/fullscreen/api/element-request-fullscreen-dialog.html?label=pr_head&max-count=1&pr=24397) I see an additional change will be needed here. When promise returned by `dialog.requestFullscreen()` should be rejected, but it turns into an unhandled promise rejection.

Handling that rejection should fix the problem.